numbers/minMax_test.go

```diff
@@ -162,7 +85,7 @@ var tripleMaxTests = []struct {
 
 func TestTripleMax(t *testing.T) {
 	for _, test := range tripleMaxTests {
-		actual := TripleMax(test.a, test.b, test.c)
```

Comment:
There might be other packages that need fixing. I remember using TripleMaxTrace in the align package, which compares scores in the 2x2 grid to decide which direction to proceed in the alignment matrix

Reply:
I don't think the tripleMaxTrace function uses the TripleMax function so it shouldn't be affected.